api/v1/helpers/nouns/users.js

```diff
@@ -23,6 +23,9 @@ module.exports = {
     POST: `Create a new ${m}`,
     PUT: `Overwrite all attributes of this ${m}`,
   },
+  scopeMap: {
+    withoutSensitiveInfo: 'withoutSensitiveInfo',
```

Comment:
The login page would fail if we remove it now. So, leave the password field in the default view for now ?